gcm_filters/kernels.py

```diff
@@ -84,9 +122,31 @@ def __call__(self, field: ArrayType):
 ALL_KERNELS[GridType.REGULAR] = RegularLaplacian
 
 
+@dataclass
+class RegularLaplacianWithArea(AreaWeightedMixin, RegularLaplacian):
+    """̵Scalar Laplacian operating on a locally orthogonal grid in three steps:
+    1) Field on locally orthogonal grid is transformed to field on regularly spaced Cartesian
+    grid with dx = dy = 1, through multiplication by cell area of original grid.
+    2) Laplacian acts on regular Cartesian grid.
+    3) Diffused field on regular Cartesian grid is transformed back to field on original grid,
+    through division by cell area of original grid.
+
+    Attributes
+    ----------
+    area: cell area of original grid
+    """
+
+    area: ArrayType
+
+    pass
+
+
+ALL_KERNELS[GridType.TRANSFORMED_TO_REGULAR] = RegularLaplacianWithArea
```

Comment:
I find this naming confusing. "Transformed" is ambiguous, and it doesn't indicate anything about the area weighting. What about`REGULAR_AREA_WEIGHTED`?

This is not a dealbreaker for me...but I'm just trying to think about what is most obvious for users.

Reply:
I actually went back and forth with the naming convention here; something similar to what you suggest here has been in the mix too! So I'm glad to hear that you find `AREA_WEIGHTED` most intuitive.